index.test.js

```diff
@@ -0,0 +1,31 @@
+var expect = require('chai').expect;
+var countdown = require('./index');
+var moment = require('moment');
+
+describe("countdown message", function() {
+  it("is a string", function() {
```

Comment:
Yes, good catch. That's not the correct description for the test. It's really testing that the message is correctly interpolating the variables. I'll revise.